.github/stale.yml

```diff
@@ -1,17 +1,25 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 60
+daysUntilStale: 90
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 7
 # Issues with these labels will never be considered stale
 exemptLabels:
   - pinned
   - security
 # Label to use when marking an issue as stale
-staleLabel: wontfix
+staleLabel: lifecycle/stale
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
-# Comment to post when closing a stale issue. Set to `false` to disable
-closeComment: false
+  any recent activity. It will be closed in 7 days if no further activity occurs. Thank you
+  for your contribution.
+
+  For more help on your issue, check out the olm-dev channel on the kubernetes slack [1] and the OLM Dev Working Group [2]
+  [1] https://kubernetes.slack.com/archives/C0181L6JYQ2
+  [2] https://docs.google.com/document/d/1Zuv-BoNFSwj10_zXPfaS9LWUQUCak2c8l48d0-AhpBw/edit?usp=sharing
```

Comment:
Is https://github.com/operator-framework/community the source of truth for this information? Maybe stalebot should link to that directly instead of enumerating channels here.

Reply:
Sure, that sounds good